website/docs/reference/datasources/graphql.md

```diff
@@ -3,496 +3,497 @@ sidebar_position: 8
 
-Configure the GraphQL API as illustrated below:
+If your API doesn't require authentication, you can select the **GraphQL API** datasource instead and skip the configuration screen.
 
-![Create GraphQL Datasource](</img/Screenshot_2022-09-07_at_11.43.59_AM.png>)<figure><figcaption align="center"><i>Create GraphQL Datasource</i></figcaption></figure>
+<figure>
+  <img src="/img/graphql-datasource-config.png" style={{width: "100%", height: "auto"}} alt="Configuring a GraphQL datasource" />
+  <figcaption align="center"><i>Configuring a GraphQL datasource.</i></figcaption>
+</figure>
 
-:::info
-All required fields are suffixed with an asterisk.
+:::tip
+If you want to connect to a local database, you can use a service like ngrok to expose it. For more information, see [How to connect to local database on Appsmith](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
 :::
 
-#### URL
-
-Use this field to add the API URL you want to access. For instance, if you want to visit [GitHub GraphQL API](https://docs.github.com/en/graphql), enter [https://api.github.com/graphql ](https://api.github.com/graphql)in the URL field.
+To connect to your database, Appsmith needs the following parameters. All required fields are suffixed with an asterisk (*).
 
-#### Headers
+**URL*:** provide the URL of the GraphQL service to query. If you are on a self-hosted instance and connecting to an API on `localhost`, use `host.docker.internal` on Windows and macOS hosts and `172.17.0.1` on Linux hosts to access services running on the host machine from within the container.
 
-HTTP Headers are an important part of the API request and response, representing the meta-data associated with the API request and response. Headers carry the following information:
+**Headers:**  provide any header key-value pairs that you'd like to include in your HTTP requests.
 
-1. Request and Response Body;
-2. Request Authorization;
-3. Response Caching;
-4. Response Cookies.
+**Query Parameters:** provide any parameters that should be passed as key-value pairs in the URL of your requests.
 
-#### Query Parameters
+**Send Appsmith signature header:** choose whether to include a special token in your request headers to help prove authenticity and integrity. For more information, read [signature headers](/core-concepts/connecting-to-data-sources/authentication/signature-header-in-api-actions).
 
-A parameter is a piece of information you provide to a query during its execution. In the query, parameters can be used independently or as a component of a larger expression to build a criterion. The API you integrate allows you to include one or more[ query parameters](/core-concepts/data-access-and-binding/querying-a-database) as part of the request.
+**Authentication Type:** choose the style of authentication to use for your queries.
```

Comment:
we also have a page for authentication type, so why not redirect there for more info like the others

Reply:
Added the link -- it seems like the content on these pages (particularly the "Basic" one) isn't super helpful, but we'll probably rehaul those pages in the future to be more valuable